css/cssom/historical.html

```diff
@@ -1,5 +1,6 @@
 <!DOCTYPE html>
 <title>Historical features</title>
+<link rel="help" href="https://drafts.csswg.org/cssom/#changes-from-5-december-2013">
```

Comment:
This is the best link I could find; it was removed from the spec and so this test is making sure that the features are not present.